google-cloud-bigtable/acceptance/bigtable/backup_test.rb

```diff
@@ -107,23 +109,35 @@
       # Wait 2 minutes so that the RestoreTable API will trigger an optimize restored table operation.
       # https://github.com/googleapis/java-bigtable/blob/33ffd938c06352108ccf7c1e5c970cce27771c72/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java#L307-L309
       sleep(120)
+
       restore_job = backup.restore restore_table_id
       _(restore_job).must_be_kind_of Google::Cloud::Bigtable::Table::RestoreJob
       restore_job.wait_until_done!
       _(restore_job.error).must_be :nil?
       restore_table = restore_job.table
       _(restore_table).must_be_kind_of Google::Cloud::Bigtable::Table
       _(restore_table.name).must_equal restore_table_id
+      _(restore_table.instance_id).must_equal instance.instance_id
 
       # optimize
       _(restore_job.optimize_table_operation_name).wont_be :nil?
       _(restore_job.optimize_table_operation_name).must_be_kind_of String
       _(restore_job.optimize_table_operation_name).must_include restore_table_id
+
+      # restore to another instance
+      restore_job = backup.restore restore_table_id, instance: instance_2
```

Comment:
Currently failing here using Ruby CI project:

```
  1) Error:
Google::Cloud::Bigtable::Table::bigtable#test_0001_creates a backup:
Google::Cloud::InvalidArgumentError: 3:Request contains an invalid argument.. debug_error_string:{"created":"@1611009167.366622000","description":"Error received from peer ipv4:142.250.68.74:443","file":"src/core/lib/surface/call.cc","file_line":1063,"grpc_message":"Request contains an invalid argument.","grpc_status":3}
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigtable-admin-v2/lib/google/cloud/bigtable/admin/v2/bigtable_table_admin/client.rb:1767:in `rescue in restore_table'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigtable-admin-v2/lib/google/cloud/bigtable/admin/v2/bigtable_table_admin/client.rb:1732:in `restore_table'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigtable/lib/google/cloud/bigtable/service.rb:768:in `restore_table'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigtable/lib/google/cloud/bigtable/backup.rb:386:in `restore'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigtable/acceptance/bigtable/backup_test.rb:128:in `block (2 levels) in <top (required)>'
```

Reply:
This acceptance test is now passing with the google-cloud-ruby CI project and my dev project.